Ladybird/BrowserWindow.cpp

```diff
@@ -30,6 +31,7 @@ BrowserWindow::BrowserWindow(Browser::CookieJar& cookie_jar, StringView webdrive
     : m_cookie_jar(cookie_jar)
     , m_webdriver_content_ipc_path(webdriver_content_ipc_path)
 {
+    qApp->installEventFilter(this);
```

Comment:
... why not installing the filter at the window level...?

Reply:
At window level I don't have accetto to tab_widget

Reply:
Use this instead (this works in line 40 after the creation of `m_tabs_container`):
```
m_tabs_container->installEventFilter(this);
``` 

This is a less "abusive" filter.

Reply:
Oh thank you :)